[2022-11-18T07:51:12.906+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: DevDataOps.task2 manual__2022-11-18T07:51:02.663580+00:00 [queued]>
[2022-11-18T07:51:13.094+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: DevDataOps.task2 manual__2022-11-18T07:51:02.663580+00:00 [queued]>
[2022-11-18T07:51:13.124+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-18T07:51:13.127+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-18T07:51:13.141+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-18T07:51:13.425+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): task2> on 2022-11-18 07:51:02.663580+00:00
[2022-11-18T07:51:13.594+0000] {standard_task_runner.py:55} INFO - Started process 262 to run task
[2022-11-18T07:51:13.725+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'DevDataOps', 'task2', 'manual__2022-11-18T07:51:02.663580+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/dev_data_ops.py', '--cfg-path', '/tmp/tmpwomyfjhm']
[2022-11-18T07:51:13.732+0000] {standard_task_runner.py:83} INFO - Job 13: Subtask task2
[2022-11-18T07:51:15.006+0000] {task_command.py:376} INFO - Running <TaskInstance: DevDataOps.task2 manual__2022-11-18T07:51:02.663580+00:00 [running]> on host d470c2e62263
[2022-11-18T07:51:15.684+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=DevDataOps
AIRFLOW_CTX_TASK_ID=task2
AIRFLOW_CTX_EXECUTION_DATE=2022-11-18T07:51:02.663580+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-18T07:51:02.663580+00:00
[2022-11-18T07:51:15.691+0000] {python.py:177} INFO - Done. Returned value was: 2
[2022-11-18T07:51:15.768+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=DevDataOps, task_id=task2, execution_date=20221118T075102, start_date=20221118T075112, end_date=20221118T075115
[2022-11-18T07:51:15.930+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-18T07:51:16.131+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
